src/main/java/cloud/eppo/EppoClient.java

```diff
@@ -40,22 +41,21 @@ public class EppoClient {
   /** @noinspection FieldMayBeFinal */
   private static EppoHttpClient httpClientOverride = null;
 
-  /** @noinspection FieldMayBeFinal */
-  private static ConfigurationStore configurationStoreOverride = null;
-
   private EppoClient(
       String apiKey,
       String sdkName,
       String sdkVersion,
       String host,
+      ConfigurationStore configurationStore,
```

Comment:
Configuration store now passed in (this is similar to other SDKs and hopefully paves the way for easier caching in Android and whatnot)